mavlink/modules/flight_input_device.py

```diff
@@ -0,0 +1,58 @@
+"""
+Creates a FlightInputDevice using dronekit which abstracts MavLink data
+Formats odometry data to dictionary 
+"""
+
+import dronekit
+from modules import drone_odometry
+
+
+class FlightInputDevice: 
+    """
+    Wrapper for flight input.
+    """
+    __create_key = object()
+
+    @classmethod
+    def create(cls, address: str) -> "tuple[bool, FlightInputDevice | None]":
+        """
+        Connection address: tcp address or serial port (e.g. "tcp:127.0.0.1:14550").
```

Comment:
Nit: Remove `Connection` . Format is `variable_name: Description of what variable_name represents/does` .